app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateReducer.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.swap.jupiter.statemanager
+
+import kotlinx.coroutines.flow.Flow
+
+interface SwapStateReducer {
```

Comment:
each reducer can work with one particular state and create new state depending on action
some states do not react to some actions